ValveResourceFormat/Resource/ResourceTypes/TextureDecompressors.cs

```diff
@@ -505,20 +505,9 @@ public static void UncompressDXT5(SKImageInfo imageInfo, BinaryReader r, Span<by
                             {
                                 if (hemiOct)
                                 {
-                                    float SignNotZero(float v)
-                                    {
-                                        return (v >= 0.0f) ? +1.0f : -1.0f;
-                                    }
-
-                                    float nx = ((data[dataIndex + 3] / 255.0f) * 2) - 1;
-                                    float ny = ((data[dataIndex + 1] / 255.0f) * 2) - 1;
+                                    float nx = ((data[dataIndex + 3] + data[dataIndex + 1]) / 255.0f) - 1.003922f;
```

Comment:
Any idea what's the magic behind `1.003922`?

Reply:
It was in the disassembly. I think this is to avoid a zero length vector.